templates/zerver/help/linking-to-zulip-website.md

```diff
@@ -0,0 +1,33 @@
+# Linking to the Zulip website
+
+Zulip sponsors free [Zulip Cloud Standard](https://zulip.com/plans/) hosting for
+hundreds of worthy organizations. Zulip has also invested into making it as easy
```

Comment:
is `worthy` the right adjective here? `qualified organizations` perhaps would sound better? not sure

Reply:
i just went with the language on https://zulip.com/plans/.

Reply:
I think the language is good. I feel like the point we're trying to make is that we're aiming to sponsor classes of organizations with positive social value to society; "qualified" just expresses we have some set of rules.

Reply:
Ah, that makes sense!